requirements.txt

```diff
@@ -0,0 +1,4 @@
+matplotlib
+tqdm
+pandas
+ads
```

Comment:
:wave: 

Should we add NumPy to this? I use it directly in a couple of places.

Possibly Jupyter is also a dependency... =P